example/README.md

```diff
@@ -1,74 +1,85 @@
 ## Example
 
-This example has a flask client and server and shows they can trace requests
-both within and between the two sites using the flask_opentracing extension and
+This example has a flask client and server and shows how to trace the requests
+to a webserver from a client using the flask_opentracing extension and
 a concrete variant of an OpenTracing tracer. To run, make sure that you run pip
-install for flask, opentracing, and lightstep (although you will not be able to
-view the lightstep traces unless you have an access token).
+install for flask, opentracing, jaeger_client.
 
-Open two terminals and navigate to this directory. Run the following commands in
+### Set up Jaeger:
+
+First, we'll have to download and run our Jaeger instance. It collects and displays
+traces in neat graphical format.
+
+If you already have Docker installed, run this:
+
+```
+docker run -d -e \
+  COLLECTOR_ZIPKIN_HTTP_PORT=9411 \
+  -p 5775:5775/udp \
+  -p 6831:6831/udp \
+  -p 6832:6832/udp \
+  -p 5778:5778 \
+  -p 16686:16686 \
+  -p 14268:14268 \
+  -p 9411:9411 \
+  jaegertracing/all-in-one:latest
```

Comment:
would it make sense to just use host networking? Most of these ports are not actually necessary for this example anyway.

Reply:
It's the way Jaeger manual presents it. I don't want to confuse people who may not be into details of Docker. In any case, ports do not hurt anyone. 
Agree?